releasenotes-builder/README.md

```diff
@@ -21,8 +21,8 @@ Jar file which includes all required dependencies will be located at
 java -jar releasenotes-builder-1.0-all.jar -localRepoPath <arg> \
      -remoteRepoPath <arg> -startRef <arg> [-endRef <arg>] \
      -releaseNumber <arg> [-outputLocation <args>] [-githubAuthToken <arg>] \
-     [-generateAll] [-generateXdoc] [-generateTwit] [-generateRss] [-generateMlist] \
-     [-xdocTemplate] [-twitterTemplate] [-rssTemplate] [-mlistTemplate] \
+     [-generateAll] [-generateXdoc] [-generateTwit] [-generateRss] [-generateMlist] [-generateGitHub] \
```

Comment:
please add Github to "generateAll"

https://github.com/checkstyle/checkstyle/pull/12008
example of execution - https://cloud.drone.io/checkstyle/checkstyle/11741/4/2

it should be generated in regular CI execution.

Reply:
Changes Done